src/mcu_opcodes.cpp

```diff
@@ -330,6 +330,23 @@ void MCU_Jump_RTS(uint8_t operand)
     mcu.pc = MCU_PopStack();
 }
 
+void MCU_Jump_RTD(uint8_t operand)
+{
+    int16_t imm = (int8_t)MCU_ReadCodeAdvance();
+    mcu.pc = MCU_PopStack();
+
+    if (operand == 0x14)
+    {
+        mcu.r[7] += imm;
+        if (mcu.r[7] & 1)
+            MCU_ErrorTrap();
+    }
+    else if (operand == 0x1c)
```

Comment:
`if` should be removed to catch any other potential cases

Reply:
Yeah but 0x1c is the word case that is not implemented, I left it with a comment for the future and also added another else at the bottom if that's ok